truss/test_data/truss_container_fs/app/common/truss_server.py

```diff
@@ -159,6 +170,48 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         _configure_logging()
 
+    def load_all(self, main_loop: IOLoop):
```

Comment:
I was anticipating us deleting this copy and pasted version of code for testing (no `truss_container_fs` folder). What do we think is our path away from this pattern?

Reply:
I'd love to get rid of this code. But the alternative seemed to be to generate this code for test run, which will turn a number of tests into integration test. Totally open to suggestions.